npm/templates/deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+exec java -jar "{DEPLOYER_PATH}" \
+    --npm_path="{NPM_PATH}" \
+    --snapshot_repo="{SNAPSHOT_REPO}" \
+    --release_repo="{RELEASE_REPO}" "$*"
```

Comment:
Pass in templated args and, at the end, positional parameters from the command line (intended to be either `-- snapshot` or `-- release`)